packages/lodestar/src/chain/chain.ts

```diff
@@ -134,6 +140,18 @@ export class BeaconChain extends (EventEmitter as { new(): ChainEventEmitter })
     return this.forkChoice.getFinalized();
   }
 
+  public async getHead(): Promise<HeadResponse> {
+    const head = this.forkChoice.head();
+    return {
+      headSlot: head.slot,
+      headBlockRoot: head.blockRoot,
+      finalizedSlot: computeStartSlotAtEpoch(this.config, head.finalizedCheckpoint.epoch),
```

Comment:
this would return incorrect value in case of skip slot on start of epoch

Reply:
No I don't think it would, it is not specced to match the block root. Checkpoint epochs match the time of the checkpoint in addition to the block root, to point to skip slots, since those need to be checkpoint-ed too. Otherwise we would only be using the block root.